uimaj-core/src/main/java/org/apache/uima/resource/impl/PearSpecifier_impl.java

```diff
@@ -38,38 +38,36 @@
   private String mPearPath;
 
   private Parameter[] mParameters;
+  private NameValuePair[] mPearParameters;
 
   /**
    * Creates a new <code>PearSpecifier_impl</code>.
    */
   public PearSpecifier_impl() {
   }
 
-  /**
-   * @return Returns the Parameters.
-   */
+  @Deprecated
   public Parameter[] getParameters() {
     return this.mParameters;
   }
+  
+  public NameValuePair[] getPearParameters() {
+    return this.mPearParameters;
+  }
 
-  /**
-   * @param parameters
-   *          The Parameters to set.
-   */
-  public void setParameters(Parameter[] parameters) {
+  @Deprecated
+  public void setParameters(Parameter... parameters) {
     this.mParameters = parameters;
   }
+  
+  public void setPearParameters(NameValuePair... pearParameters) {
+    this.mPearParameters = pearParameters;
+  }
 
-  /* (non-Javadoc)
-   * @see org.apache.uima.resource.PearSpecifier#getPearPath()
-   */
```

Comment:
What about leaving the javadoc comment but adding a Javadoc @deprecated element which specifies what do instead?

Reply:
´getPearPath´ was not deprecated. Not sure what you are referring to here.